src/components/Day/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Avatar } from "antd";
+
+import "./Day.css";
+
+function Day(props) {
+  if (props.plan !== null) {
+    return <div className="day" onClick={props.handleClick}>
+        <div className="day__side">
+          <Avatar size="large" style={{ backgroundColor: "#87d068" }}>
+            {props.plan.title[0]}
+          </Avatar>
+        </div>
+        <div className="day__main">
+          <div className="day__main--title">
+            <p>{props.day}</p>
+          </div>
+          <div className="day__main--body">
+            <p>{props.plan.title}</p>
+          </div>
+        </div>
+      </div>;
+  } else {
+    return (
+      <div className="day" onClick={props.handleClick}>
+        <p />
+        <p>{props.day}</p>
+      </div>
+    );
+  }
+}
+
+Day.propTypes = {
+  name: PropTypes.string,
```

Comment:
Missing the `plan`. The [linter](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md) can help you here.

Reply:
?? the plan gets passed from the parent component. This is a dumb component that doesn't access the store